libs/gl-client-py/Cargo.toml

```diff
@@ -18,7 +18,7 @@ crate-type = ["cdylib"]
 anyhow = { workspace = true }
 bytes = "1.4"
 env_logger = { workspace = true }
-
+futhark = { git = "https://github.com/nepet/futhark.git", branch = "main"}
```

Comment:
This dependency will prevent us from publishing to gl-client, we need to publish futhark first, then switch the dependency pointer, and only then can we upload to crates.io. Not important atm since we have other git dependencies, but I'd like to get rid of these over time, so let's not add a new one :-)

Reply:
Also I think we're now using a rune's string representation everywhere in this crate, making this dependency a bit redundant.